src/components/PageHeadingDeployment.vue

```diff
@@ -3,32 +3,23 @@
     <template #actions>
       <DeploymentToggle :deployment="deployment" @update="emit('update')" />
 
-      <template v-if="can.run.deployment">
-        <RunMenu :deployment="deployment" class="page-heading-deployment__run-menu" />
-      </template>
+      <RunMenu v-if="can.run.deployment && media.sm" :deployment="deployment" />
 
       <DeploymentMenu :deployment="deployment" @delete="handleDelete">
-        <template v-if="can.run.deployment" #additional-items>
-          <div class="page-heading-deployment__run-menu-item">
-            <router-link :to="routes.deploymentFlowRunCreate(deployment.id)">
-              <p-overflow-menu-item>
-                Run
-              </p-overflow-menu-item>
-            </router-link>
-          </div>
+        <template v-if="can.run.deployment && !media.sm" #additional-items>
+          <DeploymentQuickRunOverflowMenuItem :deployment="deployment" />
+          <DeploymentCustomRunOverflowMenuItem :deployment-id="deployment.id" />
```

Comment:
Can we adopt the `showAll` pattern from FlowRunMenu for the deployment menu as well? That way the DeploymentMenu has all its menu items and there's just a toggle to set if you want the full menu or just the concise menu.

Reply:
Totally space out on that, it simplified everything so much!

Reply:
Nice!